README.md

```diff
@@ -36,15 +36,19 @@ func Main() {
         aPIKey: os.Getenv("CONJUR_AUTHN_API_KEY"),
     )
     if err != nil {
-        // error handling
+        panic(err)
     }
     
-    secretValue, err := conjur.RetrieveSecret(variableIdentifier)
+    secretResponse, err := conjur.RetrieveSecret(variableIdentifier)
     if err != nil {
-        // error handling
+        panic(err)
     }
-    
-    fmt.Println("The secret value is: ", secretValue)
+    secretValue, err := conjur.ReadResponseBody(secretResponse)
```

Comment:
This makes sense from a security perspective, but is pretty cumbersome for the user I think. Having to make 2 calls, one make the API call and another to unwrap the secret - have you seen other API clients use this pattern?

It seems like the majority of users will want the simpler method where they `RetrieveSecret` and get the content to use. Is there a way you can think to provide both ways to use this? I wouldn't expect a function called `RetrieveSecret` return a `io.ReadCloser`, so maybe that's a way in, adding a method called `RetrieveSecretReader` that implements this more secure functionality. What do you think?

Reply:
OK I've made changes in accordance with this.